docs/manage-fulfillment-status-using-order-tags/script.liquid

```diff
@@ -1,45 +1,78 @@
 {% if event.preview %}
-  {% action "shopify" %}
-    [
-      "post",
-      "/admin/api/2019-04/orders/1234567890/fulfillments/1234567890/events.json",
-      {
-        "event": {
-          "status": "confirmed"
+  {% capture order_json %}
+    {
+      "id": 1234567890,
+      "fulfillments": [
+        {
+          "id": 23456778901,
+          "status": "open"
         }
-      }
-    ]
-  {% endaction %}
-{% else %}
+      ],
+      "tags": {{ options.shipment_tags_and_status__keyval_required | keys | first | json }}
+    }
+  {% endcapture %}
+
+  {% assign order = order_json | parse_json %}
+{% endif %}
+
+{% assign order_tags = order.tags | split: ", " %}
+
+{% if order.fulfillments == blank and options.attempt_to_fulfill_the_order_before_tagging__boolean %}
+  {% for order_tag in order_tags %}
+    {% if options.shipment_tags_and_status__keyval_required[order_tag] != blank %}
+      {% action "shopify" %}
+        [
+          "post",
+          "/admin/api/{{ task.shopify_api_version }}/orders/{{ order.id }}/fulfillments.json",
+          {
+            "fulfillment": {
+              "location_id": {{ shop.primary_location_id | json }},
+              "tracking_number": null,
+              "notify_customer": {{ options.send_shipment_notifications_to_customer__boolean }}
+            }
+          }
+        ]
+      {% endaction %}
+
+      {% break %}
+    {% endif %}
+  {% endfor %}
+
+  {% log "No shipment status tags found on this order; skipping" %}
+{% else%}
```

Comment:
While you're in here for the documentation update (if you don't mind!), can you add a space on the other end of this `else`?